docs/language/README.md

```diff
@@ -1,35 +1,268 @@
+so the example from above would be written canonically as
+```
+filter widget and price > 1000
+```
+Unlike typical log search systems, the Zed language operators are uniform:
+you can specify an operator including keyword search terms, boolean predicates,
+etc using the same syntax at any point in the pipeline.  For example,
+the predicate `count >= 10` can simply be tacked onto the output of a
+count aggregation using the filter from above and perhaps sorting
+the final about by `count` in a simple to type and edit fashion:
+```
+widget price > 1000 | count() by color | count >= 10 | sort count
+```
+The canonical form of this more complex query is:
+```
+filter widget and price > 1000
+| summarize count() by color
+| filter count >= 10
+| sort count
+```
+
+## SQL Compatiblity
+
+To encourage adoption by the vast audience of users who know and love SQL,
+a key goal of Zed is to support a superset of the data query language (DQL) portion
+of ANSI SQL.  For example, the above query can also be written in Zed as
+```
+SELECT count(), color
+WHERE widget AND price > 1000
+GROUP BY color
+HAVING count >= 10
+ORDER BY count
+```
+i.e., this SQL expression is a subset of the Zed language.
+Naturally, the SQL and Zed forms can be mixed and matched:
+```
+SELECT count(), color
+WHERE widget AND price > 1000
+GROUP BY color
+| count >= 10 | sort count
+```
+While this hybrid capability of Zed may seem questionable, our goal here
+is to have the best of both worlds: the easy interactive workflow of Zed
+combined with the ubiquity and familiarity of SQL.
+
+And because the Zed data model
+is based on a heterogenous sequence of arbitrarily typed semi-structured records,
+the Zed language is often better fit here compared to SQL.  For example, an aggregation
+that operates on heterogeneous data might look like this:
+```
+not srcip in 192.168.0.0/16
+| summarize
+    bytes := sum(src_bytes + dst_bytes),
+    maxdur := max(duration),
+    valid := and(status == "ok")
+      by srcip, dstip
+```
+This query filters out records in with `srcip` in network 192.168
+and computes three aggregations over all such records that have the `srcip` and `dstip`
+fields where some record have a `status` field, other records
+have a `duration` field and yet other records have
+`src_bytes` and `dst_bytes` fields.  Because Zed is more relaxed than SQL,
+you can throw together a bunch of related data of different types into a "data pool"
+without having to define any upfront schemas
+&mdash; let alone a schema per table &mdash;
+thereby enabling easy-to-write queries over heterogenous pools of data.
+Writing an equivalent SQL query for the different record types implied above
+would require complicated table references, nested selects, and multi-way joins.
+
+> NOTE that the SQL expression implementation is currently in prototype stage.
+> If you try it out, you may run into problems and we'd love your
+> feedback for where it breaks and how it can be improved.
```

Comment:
```suggestion
> feedback.  Feel free to [open an issue](https://github.com/brimdata/zed/issues/new)
> or come talk to us on our [public Slack](https://www.brimsecurity.com/join-slack/)
> about where you've seen it break or where you think it can be improved.
```